bin/rails

```diff
@@ -1,11 +1,7 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-begin
-  load File.expand_path('spring', __dir__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```

Comment:
dumb question:  I don't think this exception handling was part of rails boilerplate before so ... do we want to keep it?

Reply:
No idea. However, I can't think of any reason that this app would need special handling here.

Reply:
This was just what rails generated when we did `rails new`. I don't think there's any reason not to accept this.